### PR TITLE
Add Meterpreter compatibility matrix generation

### PR DESCRIPTION
**Pull request by adfoster-r7**
Add Meterpreter compatibility matrix generation to Github's acceptance test runs. It's a separate effort to get this onto docs.metasploit.com automagically

It's inside the generated final report for now:

image

And opening:

image

image

image

## Verification

Download the generated report

### CODE REVIEW

File: .github/workflows/acceptance.yml [hunk 1]
```diff
@@ -180,13 +203,22 @@ jobs:
       - name: allure generate
         if: always()
         run: |
+          ls -la
```
Comment (bwatters-r7) [inline .github/workflows/acceptance.yml:206]: Delete

File: spec/acceptance/README.md [hunk 2]
```diff
@@ -17,7 +20,7 @@ Running Meterpreter test suite:
 SPEC_OPTS='--tag acceptance' bundle exec rspec './spec/acceptance/meterpreter_spec.rb'
 ```
 
-Skip loading of Rails/Metasplotit with:
+Skip loading of Rails/Metasplot with:
```
Comment (jmartin-tech) [inline spec/acceptance/README.md:23]:
```suggestion
Skip loading of Rails/Metasploit with:
```

**Comment by cgranleese-r7** [comment]
Everything appears to be working as expected. Happy to land once all goes green 👍 
image

**Comment by adfoster-r7** [comment]
## Release Notes

Add Meterpreter compatibility matrix generation to Github's acceptance test runs. Now it's possible to see visually which Meterpreters support particular functionality.

**Pull request merged**